exercises/concept/amusement-park/amusement-park.spec.js

```diff
@@ -0,0 +1,197 @@
+import {
+  createVisitor,
+  revokeTicket,
+  ticketStatus,
+  simpleTicketStatus,
+  gtcVersion,
+} from './amusement-park';
+
+describe('createVisitor', () => {
+  test('correctly assembles the visitor object', () => {
+    const expected1 = { name: 'Imran Kudrna', age: 21, ticketId: 'ECMZR67C' };
+    const expected2 = { name: 'Nālani Sansone', age: 76, ticketId: 'YX52GB06' };
+    const expected3 = { name: 'Bogiła Kalš', age: 55, ticketId: '30Z2OKJP' };
```

Comment:
Haha nice. Are these from that name generator?

Reply:
yes, the one the Sascha recommended ([link](https://www.behindthename.com/random/?number=1&sets=5&gender=u&surname=&randomsurname=yes&norare=yes&usage_afr=1&usage_alb=1&usage_ara=1&usage_arm=1&usage_aze=1&usage_bas=1&usage_bre=1&usage_bul=1&usage_cat=1&usage_chi=1&usage_cro=1&usage_cze=1&usage_dan=1&usage_dut=1&usage_est=1&usage_fin=1&usage_fre=1&usage_fri=1&usage_gal=1&usage_geo=1&usage_ger=1&usage_gre=1&usage_haw=1&usage_heb=1&usage_hun=1&usage_ice=1&usage_igb=1&usage_ind=1&usage_ins=1&usage_iri=1&usage_ita=1&usage_jap=1&usage_kaz=1&usage_khm=1&usage_kor=1&usage_lat=1&usage_lim=1&usage_lth=1&usage_mac=1&usage_mao=1&usage_ame=1&usage_nor=1&usage_occ=1&usage_per=1&usage_pol=1&usage_por=1&usage_rmn=1&usage_rus=1&usage_sco=1&usage_ser=1&usage_slk=1&usage_sln=1&usage_spa=1&usage_swe=1&usage_tha=1&usage_tur=1&usage_ukr=1&usage_urd=1&usage_vie=1&usage_wel=1&usage_yor=1))